api-client/src/maps/index.ts

```diff
@@ -46,3 +52,9 @@ export const defaultViewportConfig: MapViewViewport = {
 
 export const detailZoom = 15
 export const locationName = 'name'
+
+export const mapMarker: MarkerConfig = {
+  iconSize: 1.75,
+  symbol: 'marker_15',
```

Comment:
Where do these names/makers come from?

Reply:
they are coming from sprite json
https://maps.tuerantuer.org/styles/integreat/sprites.json
to check icons you can got to the repo and check svgs
https://github.com/digitalfabrik/integreat-maplibre-style/tree/gh-pages/svgs/svgs_iconset